examples/vision/sample_size_estimate.py

```diff
@@ -0,0 +1,474 @@
+"""
+Title: Estimating required sample size for model training
+Author: [JacoVerster](https://twitter.com/JacoVerster)
+Date created: 2021/05/20
+Last modified: 2021/06/05
+Description: Estimate the model accuracy for a specific training set size.
```

Comment:
This is the reverse of the title... how about "Modeling the relationship between training set size and model accuracy."

Reply:
I was also contemplating this, good suggestion thanks.